spec/requests/home_page_spec.rb

```diff
@@ -4,7 +4,7 @@
   describe "Home page" do
     it "should have the name of the app" do
       visit root_path
-      expect(page).to have_content('User Registrations App')
+      expect(page).to have_content('Expenses App')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.